src/Components/Modal/ModalWrapper.tsx

```diff
@@ -0,0 +1,171 @@
+import React from "react"
+import styled, { injectGlobal, keyframes } from "styled-components"
+import FadeTransition from "../Animation/FadeTransition"
+import { media } from "../Helpers"
+import { CtaProps } from "./ModalCta"
+
+export interface ModalWrapperProps extends React.HTMLProps<ModalWrapper> {
+  blurContainerSelector?: string
+  cta?: CtaProps
+  onClose?: () => void
+  isWide?: boolean
+  image?: string
+  show?: boolean
+}
+
+export interface ModalWrapperState {
+  isAnimating: boolean
+  isShown: boolean
+  blurContainers: Element[]
+}
+
+injectGlobal`
+  .blurred {
+    filter: blur(50px);
+  }
+`
+
+export class ModalWrapper extends React.Component<
+  ModalWrapperProps,
+  ModalWrapperState
+> {
+  static defaultProps = {
+    show: false,
+    blurContainerSelector: "",
+  }
+
+  state = {
+    isAnimating: this.props.show || false,
+    isShown: this.props.show || false,
+    blurContainers: this.props.blurContainerSelector
+      ? Array.from(document.querySelectorAll(this.props.blurContainerSelector))
+      : [],
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (this.props.show !== nextProps.show) {
+      this.setState({
+        isAnimating: true,
+        isShown: nextProps.show,
+      })
+    }
+  }
+
+  componentWillUnmount() {
+    this.removeBlurToContainers()
+  }
+
+  close = () => {
+    this.props.onClose()
+    this.removeBlurToContainers()
+  }
+
+  addBlurToContainers = () => {
+    for (const container of this.state.blurContainers) {
+      container.classList.add("blurred")
+    }
+  }
+
+  removeBlurToContainers = () => {
+    for (const container of this.state.blurContainers) {
+      container.classList.remove("blurred")
+    }
+  }
+
+  render(): JSX.Element {
+    const { children, isWide, image } = this.props
+    const { isShown, isAnimating } = this.state
+
+    if (isShown) {
+      this.addBlurToContainers()
+    } else {
+      this.removeBlurToContainers()
+    }
+
+    return (
+      <Wrapper isShown={isShown || isAnimating}>
```

Comment:
I'm assuming that the Modal maybe be rendered in a portal which might be the reason why it's not being recognized as being rendered in the theme provider. To combat that, wrap this in the `Theme` component from palette. The color should work fine after that.